googletest/src/gtest.cc

```diff
@@ -6282,6 +6417,21 @@ static const char kColorEncodedHelpMessage[] =
     "'*'\n"
     "      matches any substring; ':' separates two patterns.\n"
     "  @G--" GTEST_FLAG_PREFIX_
+    "tag=@YPOSTIVE_PATTERNS"
+    "[@G-@YNEGATIVE_PATTERNS]@D\n"
+    "      Run only the tests whose tag matches one of the positive "
+    "patterns but\n"
+    "      none of the negative patterns. '?' matches any single character; "
+    "'*'\n"
+    "      matches any substring; ':' separates two patterns.\n"
+    "  @G--" GTEST_FLAG_PREFIX_
+    "size=@YPOSTIVE_PATTERNS"
+    "[@G-@YNEGATIVE_PATTERNS]@D\n"
+    "      Run only the tests whose size matches one of the positive "
+    "patterns but\n"
+    "      none of the negative patterns. "
+    "'S'\n"
```

Comment:
Is this showing in the help that `S` is the default?

Reply:
Should be fixed